src/screenComponents/combatManeuver.cpp

```diff
@@ -3,12 +3,133 @@
 #include "combatManeuver.h"
 #include "powerDamageIndicator.h"
 #include "snapSlider.h"
-
+#include "preferenceManager.h"
 #include "gui/gui2_progressbar.h"
 
 GuiCombatManeuver::GuiCombatManeuver(GuiContainer* owner, string id)
 : GuiElement(owner, id)
 {
+	static std::vector<std::pair<string, sf::Keyboard::Key> > sfml_key_names = {
```

Comment:
This looks like a copy&paste from the hotkey code. To prevent duplication the hotkey class could have a static function (or 2) to convert keynames from/to string.

Reply:
Yes it is. I thought about making that vector global in some way, but figured it could be a bit overkill for using it in just one file.
Will look into do a conversion function.

Reply:
It's never overkill to remove code duplication. (Not saying the whole hotkey part is very well designed anyhow)